### ISSUE TITLE
Find All References pop up is broken

### ISSUE THREAD

**Issue by Mellbourn** [Microsoft/vscode#58610]
- VSCode Version: 1.27.1
- OS Version: macOS 10.13.6

Steps to Reproduce:

1. Open a JavaScript project
2. Open a file containing a definition
3. Position the cursor on a definition
4. Press Shift-F12 to find all references
5. References are found, but not displayed properly. The pop up is empty and compressed in height:

![find all references is shown empty](https://user-images.githubusercontent.com/1785727/45497205-83801280-b777-11e8-83a0-68263602a7c9.png)

Does this issue occur when all extensions are disabled?: Yes
It also occurs if I remove all settings. 
I have tried reinstalling VSCode, it did not help.
It does not occur in the current insiders build (1.28.0 31bf4ccab388).

**Comment by mjbvz** [comment]
Closing since you said that this is fixed in insiders. Let me know if you see this in insiders as well. The current insiders build will become the next stable build at the end of this month

**Comment by Mellbourn** [comment]
Upgrading to the next version of code did not work. I did solve this, though:

I very slowly slid the mouse pointer vertically over the screen. When the mouse pointer was clearly *above* the reference dialog (not "on top of", but higher up on the screen than the highest location of the reference dialog), it briefly turned into a resize handler. At that point, I pressed the mouse button down, and by dragging it downward, I could resize the window to a normal size.

So, it seems that I, in some unknown way, had managed to resize the dialog so that the height became negative.
